src/services/iamRole/format.ts

```diff
@@ -24,7 +24,8 @@ export default ({
     RoleLastUsed,
     AssumeRolePolicyDocument: assumeRolePolicy = '',
     MaxSessionDuration: maxSessionDuration = 0,
-    Policies: inlinePolicies = [],
+    InlinePoliciesName: inlinePolicies = [],
+    InlinePoliciesDocuments: inlineFormattedPolicies,
```

Comment:
I would just make this one object. `inlinePolicies` which is a array of obj [{ id: ..., name: ..., content: formatIamJsonPolicy()}]`